OutfitterInventory.lua

```diff
@@ -1,3 +1,9 @@
+-- Local functions to fix 3.4.1 changes
+local GetContainerNumSlots = C_Container and C_Container.GetContainerNumSlots or _G.GetContainerNumSlots
+local GetContainerItemLink = C_Container and C_Container.GetContainerItemLink or _G.GetContainerItemLink
+local GetContainerItemInfo = C_Container and C_Container.GetContainerItemInfo or _G.GetContainerItemInfo

```

Comment:
This fix is not enough for me.
line 88 (before fix) : 
`itemInfo.Gem1Link = select(2, GetItemInfo(itemInfo.Gem1))`
will proc the following error : 
```
Date: 2023-01-18 12:41:33
ID: 6
Error occured in: Global
Count: 1
Message: Interface/AddOns/Outfitter/OutfitterInventory.lua line 88:
   Usage: GetItemInfo(itemID|"name"|"itemlink")
Debug:
   [string "=[C]"]: ?
   [string "=[C]"]: GetItemInfo()
   [string "@Interface/AddOns/Outfitter/OutfitterInventory.lua"]:88: GetBagItemInfo()
   [string "@Interface/AddOns/Outfitter/OutfitterInventory.lua"]:837: Synchronize()
   [string "@Interface/AddOns/Outfitter/OutfitterInventory.lua"]:748: GetInventoryCache()
   [string "@Interface/AddOns/Outfitter/Outfitter.lua"]:4019: GetNewItemsOutfit()
   [string "@Interface/AddOns/Outfitter/OutfitterEquipment.lua"]:981: RestoreSavedStack()
   [string "@Interface/AddOns/Outfitter/OutfitterEquipment.lua"]:959: Initialize()
   [string "@Interface/AddOns/Outfitter/Outfitter.lua"]:4779: Function()
   [string "@Interface/AddOns/Outfitter/Libraries/MC2SchedulerLib/MC2SchedulerLib.lua"]:242: OnUpdate2()
   [string "@Interface/AddOns/Outfitter/Libraries/MC2SchedulerLib/MC2SchedulerLib.lua"]:178: OnUpdate()
   [string "@Interface/AddOns/Outfitter/Libraries/MC2SchedulerLib/MC2SchedulerLib.lua"]:20:
      ...fitter/Libraries/MC2SchedulerLib/MC2SchedulerLib.lua:20
Locals:

AddOns:
  Swatter, v3.4.6837 (SwimmingSeadragon)
  WowheadLooter, v30401
  AddonUsage, v3.1.2
  Atlas, vv1.52.05
  AtlasBattlegrounds, vv1.52.01
  AtlasBurningCrusade, vv1.52.01
  AtlasClassicWoW, vv1.52.01
  AtlasDungeonLocs, vv1.52.01
  AtlasTransportation, vv1.52.01
  AtlasWrathoftheLichKing, vv1.52.01
  AtlasLootClassic, vv3.0.7
  AtlasLootClassicDungeonsAndRaids, vv3.0.7
  Attune, v311
  AucAdvanced, v3.4.6844 (SwimmingSeadragon)
  AucFilterBasic, v3.4.6803 (SwimmingSeadragon)
  AucStatHistogram, v3.4.6805 (SwimmingSeadragon)
  AucStatiLevel, v3.4.6809 (SwimmingSeadragon)
  AucStatPurchased, v3.4.6806 (SwimmingSeadragon)
  AucStatSimple, v3.4.6807 (SwimmingSeadragon)
  AucStatStdDev, v3.4.6808 (SwimmingSeadragon)
  AucUtilFixAH, v3.4.6810 (SwimmingSeadragon)
  BagBrother, v
  Bagnon, v10.0.15
  Bartender4, v4.14.4
  BeStride, v2.0.10
  BigWigs, vv46.1-classic
  Bistooltip, v7.13
  CEPGP, v
  CEPGPWotLK, v1.0.9
  ClassicHealPrediction, vv1.8.3
  Coordinates, v2.4
  DeathNote, v30400.5-release
  Decursive, v2.7.8.13
  Details, v#Details.DF.Wrath.10407.148
  DetailsCompare2, v
  DetailsEncounterDetails, v
  DetailsRaidCheck, v
  DetailsTinyThreat, v
  DetailsVanguard, v
  Enchantrix, v3.4.6849 (SwimmingSeadragon)
  ExtendedCharacterStats, v3.0.8
  GatherMate2, v1.47-classic
  GTFO, v5.0
  Informant, v3.4.6850 (SwimmingSeadragon)
  ItemRack, v
  LeatrixMaps, v3.0.83
  LeatrixPlus, v3.0.83
  LockoutTracker, v1.0.4
  LoggerHeadLite, v1.6.3
  Masque, v10.0.4
  MRT, v4720
  NeatPlates, vv429
  NeatPlatesBlizzardPlates, v2.0
  NeatPlatesClassicPlates, v
  NeatPlatesHub, v
  NeatPlatesWidgets, v
  NovaInstanceTracker, v1.40
  NovaWorldBuffs, v2.43
  Outfitter, v3.0.2
  PallyPower, vv1.5.1-classic
  ProfessionMaster, v1.6.1
  Questie, v7.4.7
  RareScanner, v3.4.1
  Scrap, v10.0.9
  Stubby, v3.4.6816 (SwimmingSeadragon)
  TacoTip, v0.3.5
  TitanClassic, v1.5.4.30401
  TitanClassicAmmo, v1.5.4.30401
  TitanClassicBag, v1.5.4.30401
  TitanClassicClock, v1.5.4.30401
  TitanClassicGold, v1.5.4.30401
  TitanClassicLocation, v1.5.4.30401
  TitanClassicLootType, v1.5.4.30401
  TitanClassicPerformance, v1.5.4.30401
  TitanClassicRegen, v1.5.4.30401
  TitanClassicRepair, v1.5.4.30401
  TitanClassicVolume, v1.5.4.30401
  TitanClassicXP, v1.5.4.30401
  TotemTimers, v3.1.13
  TrinketMenu, v9.0.8
  VendorPrice, v1.6.3
  WeakAuras, v5.3.4
  WeakAurasCompanion, v1.1.0
  BlizRuntimeLib_enUS v3.4.1.30401 <none>
  (ck=977)
```
Adding print(itemInfo.Gem1) outputs `table: 000001C0774BBF30`